plugins/common/http/config.go

```diff
@@ -2,16 +2,16 @@ package httpconfig
 
 import (
 	"context"
-	"net/http"
-	"time"
-
+	"fmt"
 	"github.com/benbjohnson/clock"
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/plugins/common/cookie"
 	oauthConfig "github.com/influxdata/telegraf/plugins/common/oauth"
 	"github.com/influxdata/telegraf/plugins/common/proxy"
 	"github.com/influxdata/telegraf/plugins/common/tls"
+	"net/http"
+	"time"
```

Comment:
Can you please order the import by

1. built-in libraries such as `fmt`, `net/http`, `time`, etc
2. 3rd party libraries e.g. everything starting with ` github.com/...` except for telegraf internal ones
3. telegraf internal libraries, i.e. everything starting with `github.com/influxdata/telegraf`

Separate all those groups by a newline and sort alphabetically within each group.

Reply:
I found a handy setting in Goland for this request. Reordered!